src/AccessibilityInsights.SharedUx/Controls/CustomControls/AutomatedChecksCustomListControl.xaml.cs

```diff
@@ -347,7 +347,9 @@ private void lviResults_PreviewKeyDown(object sender, KeyEventArgs e)
             }
         }
 
+#pragma warning disable CA1822 // Analyzer incorectly thinks this can be static
```

Comment:
There are a lot of false positives for this rule; I think the downside of wasting time considering a false positive probably outweighs the benefit provided by this rule, so I'd be inclined to just disable the rule until we have a fix for the false positives. They look like the issue tracked by https://github.com/dotnet/roslyn-analyzers/issues/4733 - I'm not sure when exactly we'd expect to receive a fix for it, though.